### PR TITLE
Use the UCR DB for all new UCRs

### PR DESCRIPTION
**Pull request by czue**
Existing reports will be grandfathered in.

Has a migration but it should be quick.

@snopoke cc @benrudolph

### CODE REVIEW

File: corehq/apps/userreports/models.py [hunk 1]
```diff
@@ -232,10 +232,12 @@ def by_domain(cls, domain):
         )
 
     @classmethod
+    def all_ids(cls):
+        return [res['id'] for res in cls.get_db().view('userreports/data_sources_by_build_info',
+                                                       reduce=False, include_docs=False)]
+    @classmethod
```
Comment (dimagimon) [inline corehq/apps/userreports/models.py]: E301 expected 1 blank line, found 0

**Comment by snopoke** [comment]
Looks good

**Comment by sheelio** [comment]
@czue 

Was setting up a new dev instance of HQ today (blank couch database) and I think this was causing an error when running
./manage.py syncdb -noinput

I think it was causing an exception because the userreports view didn't exist at all yet:
https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/userreports/migrations/0001_initial.py#L16

**Comment by czue** [comment]
thanks and makes sense. will fix that.

**Comment by czue** [comment]
should be fixed in https://github.com/dimagi/commcare-hq/pull/8340

**Pull request merged**